runtime/drivers/admin/admin.go

```diff
@@ -124,7 +124,9 @@ type Handle struct {
 	gitURLExpiresOn time.Time
 
 	// downloadURL is set when using one-time uploads
-	downloadURL string
+	downloadURL      string
+	ArchiveID        string
+	ArchiveCreatedOn time.Time
```

Comment:
nit: first letter should be lowercase for private fields (that are not accessed outside of the package)